cmd/health_alive.go

```diff
@@ -0,0 +1,26 @@
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/ory/oathkeeper/sdk/go/oathkeeper/client/api"
+	"github.com/ory/x/cmdx"
+	"github.com/spf13/cobra"
+)
+
+// healthCmd represents the health command
+var healthAliveCmd = &cobra.Command{
+	Use:   "alive",
+	Short: "Command for checking alive status",
+	Run: func(cmd *cobra.Command, args []string) {
+		client := newClient(cmd)
+
+		r, err := client.API.IsInstanceAlive(api.NewIsInstanceAliveParams())
+		cmdx.Must(err, "%s", err)
+		fmt.Println(cmdx.FormatResponse(r.Payload))
```

Comment:
I added explicit exit codes in all cases now. `cmdx.Must(err, "%s", err)` will exit 1 if there was an error e.g. a 404 response. If no error, it will exit 1 if `r.Payload.Status != "ok"`. Otherwise, exit 0

Reply:
Ah right, I forgot that the API will return an error if the code is not 200. We should still print out the received status code, which will make it easier to debug :)

Reply:
The error status code is printed in the error message. For example:
```
$ oathkeeper health -e http://localhost:4456/bad ready
> unknown error (status 404): {resp:0xc0028aee10}
```
I can't find any way to get a reference to the StatusCode based on the what the internal API client returns.

Reply:
You can type assert the error with `runtime.APIError` and then you have access to the Response, code, and so on, so e.g.

```
if apiErr, ok := err.(*runtime.APIError); ok {
// ...
} else {
// ...
}
```

Reply:
however, I think it's ok as it is at the moment!